src/handlers/transfers/FxFulfilService.js

```diff
@@ -78,23 +87,39 @@ class FxFulfilService {
   async validateHeaders({ transfer, headers, payload }) {
     let fspiopError = null
 
-    if (headers[SOURCE]?.toLowerCase() !== transfer.counterPartyFspName.toLowerCase()) {
+    if (
+      headers[SOURCE]?.toLowerCase() !==
+      transfer.counterPartyFspName.toLowerCase()
+    ) {
       fspiopError = fspiopErrorFactory.fxHeaderSourceValidationError()
     }
-    if (headers[DESTINATION]?.toLowerCase() !== transfer.initiatingFspName.toLowerCase()) {
+    if (
+      headers[DESTINATION]?.toLowerCase() !==
```

Comment:
@geka-evk I hope the linter hook is not automatically reformatting files on `commit` or `push`. Will try and debug that to prevent auto reformatting in the future.